kiwi_keg/keg.py

```diff
@@ -39,10 +40,14 @@
         Destination directory for generated description, default cwd
 
     --disable-root-tar
-        Option to disable the creation of a tarball root.tar.gz in destination directory.
-        If present, an overlay tree would be created instead.
+        Option to disable the creation of root.tar.gz in destination directory.
+        If present, an overlay tree will be created instead.
         [default: false]
 
+    --dump-dict
+        Dump generated data dictionary to stdout instead of generating an image
+        description. Useful for debugging.
+
```

Comment:
It is probably useful to have the generated image definition and the debug output, I think there is no reason to have them mutually exclusive. Of course one can run keg and generate the image definition and the run keg again in debug mode, if that is the intention I am OK with that.

Reply:
I implemented it this way because I wanted it to dump the dictionary to stdout rather than logging it as a debug message, so you could e.g. feed it to another tool, and I felt that makes it more of a different operating mode. I used it mostly to verify that the data structures where as intended while working on the code, and didn't really care about the generation part at that point. If people prefer that it can do both things in one go I can change it. I don't expect it to be very useful outside development though.